device-types/Cisco/MDS-C9148S.yaml

```diff
@@ -0,0 +1,138 @@
+---
+manufacturer: Cisco
+model: MDS 9148S
+slug: ds-c9148s
```

Comment:
The slug should have `cisco-` prepended to it

Reply:
The slug has been prepended with "cisco-".